Storage/src/Connection/RetryTrait.php

```diff
@@ -86,6 +86,9 @@ trait RetryTrait
         'objects.update' => ['ifMetagenerationMatch']
     ];
 
+    public static $RETRY_STRATEGY_ALWAYS = "always";
```

Comment:
A small comment to explain these will go a long way.

Reply:
+1, yes please add docs. It would also be helpful to know how users are expected to set these values and whether it's per-client or per-call

Reply:
Yeah, done that.